dygraph/mobilenet/train.py

```diff
@@ -116,14 +119,36 @@ def train_mobilenet():
             optimizer.set_dict(opti_dict)
 
         # 3. reader
-        train_data_loader = utility.create_data_loader(is_train=True, args=args)
+        #train_data_loader = utility.create_data_loader(is_train=True, args=args)
         test_data_loader = utility.create_data_loader(is_train=False, args=args)
         num_trainers = int(os.environ.get('PADDLE_TRAINERS_NUM', 1))
         imagenet_reader = reader.ImageNetReader(seed=0, place_num=place_num)
-        train_reader = imagenet_reader.train(settings=args)
-        test_reader = imagenet_reader.val(settings=args)
-        train_data_loader.set_sample_list_generator(train_reader, place)
-        test_data_loader.set_sample_list_generator(test_reader, place)
+        #train_reader = imagenet_reader.train(settings=args)
+        #test_reader = imagenet_reader.val(settings=args)
+        #train_data_loader.set_sample_list_generator(train_reader, place)
+        #test_data_loader.set_sample_list_generator(test_reader, place)
+
+        train_dataset = ImageNetDataset(
+            os.path.join(args.data_dir, "train"), mode='train')
+
+        train_data_loader = DataLoader(
+            train_dataset,
+            batch_size=args.batch_size,
+            places=place,
+            shuffle=True,
+            drop_last=True,
+            num_workers=10)
```

Comment:
这个用多进程一般情况下docker应该直接会崩，CE会不会被卡？或者要不要加点注释提醒下用户？虽然文档和报错里也都有，但QA可能会因为这个问题来找

例如：

#### Note:
- Docker `/dev/shm` size: DataLoader use mmap(Memory Map) to implement shared memory. You need to confirm if your shared memory is big enough, otherwise a bus error will occur. The docker default `/dev/shm` size is 64M, you can set it by docker `--shm-size` parameter, for example, `docker run --name [Name of container] -it --shm-size=2g -v $PWD:/paddle <imagename> /bin/bash`, you can see docker's shared memory size by `df -h`
- You need to make sure your `/dev/shm` size is larger than the following calculation results, otherwise, it will report an `Bus error` because of insufficient shared memory
```
DataLoader Num * queue capacity * 1 batch data size
```

Reply:
好 我改下